content/encrypt-extra.md

```diff
@@ -18,17 +18,17 @@ tableOfContents: true
 
 The <u>Disks</u> application can be used to encrypt the extra drive and it comes pre-installed on Pop!_OS and Ubuntu.
 
-In this screenshot the extra drive can be seen and then select the **Gear icon** under Volumes. Then select the **Format Partition...** option to open the 'Format Volume' window.
+In this screenshot, the extra drive can be seen. Select the **Gear icon** under Volumes. Then select the **Format Partition...** option to open the 'Format Partition' dialog.
```

Comment:
The title on the dialog is "Format Volume", even though the option to open it is "Format Partition", so we might want to leave that change out.